common/artifacts/00_artifacts.txt

```diff
@@ -1599,11 +1596,11 @@ ruby_bracelets = {
 }
 
 ruby_bracelets_2 = {
-	quality = 3
-	monthly_character_prestige = 0.7
-	castle_opinion = 4
-	tribal_opinion = 3
-	town_opinion = 3
+	quality = 4
```

Comment:
Quality changed, is it ok?

Reply:
Hmm, something went wrong... I'll check it all again tonight or tomorrow and ping you again...

Reply:
Okay, apparently I saw wrong column on my smart phone earlier. I changed quality from 3 to 4 so it would fit better with other artifacts, too. All other <artifact_xy_2> are tier 4 instead of 3. Don't know if this one was intended by PDX, but probably it was just overlooked.

Reply:
https://forum.paradoxplaza.com/forum/index.php?threads/ck-ii-2-8-3-2-the-artifact-ruby_bracelets_2-should-be-quality-4.1109030/